app/views/trn_submissions/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
```

Comment:
not sure why we have a `grid-row` & `grid-column-full` inside a `grid-row` & `grid-column-full`

Reply:
Neither am I. As discussed offline this will get changed to use the Panel component so we can sort it out in that PR.